devices/Mpu6886/Mpu6886.cs

```diff
@@ -0,0 +1,176 @@
+            Thread.Sleep(1);
+
+            // USER_CTRL(0x6a)
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.UserControl, 0x00 }));
+            Thread.Sleep(1);
+
+            // FIFO_EN(0x23)
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.FifoEnable, 0x00 }));
+            Thread.Sleep(1);
+
+            // INT_PIN_CFG(0x37)
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.IntPinBypassEnabled, 0x22 }));
+            Thread.Sleep(1);
+
+            // INT_ENABLE(0x38)
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.InteruptEnable, 0x01 }));
+            Thread.Sleep(100);
+
+            //setGyroFsr
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.GyroscopeConfiguration, 3 << 3 }));
+            Thread.Sleep(10);
+
+            _i2c.Write(new SpanByte(new byte[] { (byte)Mpu6886.Register.AccelerometerConfiguration1, 2 << 3 }));
+            Thread.Sleep(10);
+        }
+
+        private Vector3 GetRawAccelerometer()
+        {
+            SpanByte vec = new byte[6];
+            Read(Mpu6886.Register.AccelerometerMeasurementXHighByte, vec);
+
+            var x = (vec[0] << 8 | vec[1]);
+            var y = (vec[2] << 8 | vec[3]);
+            var z = (vec[4] << 8 | vec[5]);
+
+            return new Vector3(x, y, z);
+        }
+
+        private Vector3 GetRawGyroscope()
+        {
+            SpanByte vec = new byte[6];
+            Read(Mpu6886.Register.GyropscopeMeasurementXHighByte, vec);
+
+            var x = (vec[0] << 8 | vec[1]);
+            var y = (vec[2] << 8 | vec[3]);
+            var z = (vec[4] << 8 | vec[5]);
+
+            return new Vector3(x, y, z);
+        }
+
+        private int GetRawTemperature()
+        {
+            SpanByte vec = new byte[2];
+            Read(Mpu6886.Register.TemperatureMeasurementHighByte, vec);
+
+            return (vec[0] << 8 | vec[1]); ;
+        }
+
+        /// <summary>
+        /// Reads the current accelerometer values from the registers, and compensates them with the accelerometer resolution.
+        /// </summary>
+        /// <returns>Vector of acceleration</returns>
+        public Vector3 GetAccelerometer() => GetRawAccelerometer() * accelerometerResolution;
+
+        /// <summary>
+        /// Reads the current gyroscope values from the registers, and compensates them with the gyroscope resolution.
+        /// </summary>
+        /// <returns>Vector of the rotation</returns>
+        public Vector3 GetGyroscope() => GetRawGyroscope() * gyroscopeResolution;
+
+        /// <summary>
+        /// Reads the register of the on-chip temperature sensor which represents the MPU-6886 die temperature.
+        /// </summary>
+        /// <returns>Temperature in degrees Celcius</returns>
+        public double GetTemperature() => GetRawTemperature() / 326.8 + 25.0; // p43 of datasheet describes the room temp. compensation calcuation
```

Comment:
Please use UnitsNet for Temperature. So anyone can then consume it the way they like.

Reply:
Checked, according to the documentation it's -25.0, not +25 ;-)

Reply:
Unitsnet added. 
Can we double check the offset @Ellerbach ? I know Arduino libraries are not the best to look at, but the [M5 Stack library also does +25](https://github.com/m5stack/M5StickC/blob/3e00ecfa0897a432995a3f80874715cbe6ad60ee/src/utility/MPU6886.cpp#L336). And for me the datasheet is not super clear on this. Happy to follow your lead, if you are still convinced. :)

Reply:
Run it and if the room temperature is about the same as you read, then offset is in the right direction. 64°C is clearly not the right temperature for a sensor like this. So I would personally give it a try with the -25C and see how it goes.

Reply:
Switching to -25 results in temperature of 13C (even for Belgium that's cold 😁). So I'll leave the calculation as is. The datasheet says it's the chip's "die temperature", and my device feels pretty warm in hand, so I'm willing to believe the die temp is high.

Reply:
Very fair enough!